app/controllers/webhooks/stripe/base_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Webhooks::Stripe::BaseController < ApplicationController
+  include StripeWebhookEvents
+
+  skip_before_action :verify_authenticity_token
```

Comment:
You don't need to `skip_before_action` _and_ change `protect_from_forgery`. Just one will do.